src/FixerDefinition/VersionSpecification.php

```diff
@@ -41,8 +41,8 @@ public function __construct(?int $minimum = null, ?int $maximum = null)
             throw new \InvalidArgumentException('Minimum or maximum need to be specified.');
         }
 
-        if (null !== $minimum && 1 > $minimum) {
-            throw new \InvalidArgumentException('Minimum needs to be either null or an integer greater than 0.');
+        if (null !== $minimum && 7_04_00 >= $minimum) { // @TODO: update condition when lowest supported PHP version is changed
```

Comment:
i wonder if we can have this value as namespaced constant somewhere.
We use it also in `.php-cs-fixer.php-lowest.php` and `php-cs-fixer` files

Reply:
I was checking that too, the problem is that in `php-cs-fixer` value is used before Composer's autoloader jumps and I guess we want to keep it that way.